sky/execution.py

```diff
@@ -162,26 +161,18 @@ def _execute(
         if dryrun.
     """
     cluster_exists = False
```

Comment:
unused; L166 too

Reply:
It is used by other part of our code. Revert it to the original place.